src/client/components/forms/profile.js

```diff
@@ -67,66 +87,168 @@ class ProfileForm extends React.Component {
-					label="Area"
-					labelClassName="col-md-4"
-					placeholder="Select area..."
-					ref={(ref) => this.area = ref}
-					select2Options={select2Options}
-					wrapperClassName="col-md-4"
-				/>
-				<div className="form-group">
-					<div className="col-md-4 col-md-offset-4">
-						<Button
-							block
-							bsSize="large"
-							bsStyle="primary"
-							type="submit"
+			<Grid>
+				<h1>Edit Profile</h1>
+				<Row>
+					<Col md={12}>
+						<p className="lead">Edit your public profile.</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col
+						id="profileForm"
+						md={6}
+						mdOffset={3}
+					>
+						<form
+							className="form-horizontal"
+							onSubmit={this.handleSubmit}
 						>
-							Update!
-						</Button>
-					</div>
-				</div>
-			</form>
+							{loadingElement}
+							<Row>
+								<Col
+									md={4}
+									mdOffset={2}
+								>
+									<Input
+										defaultValue={initialDisplayName}
+										label="Display Name"
+										ref={(ref) => this.name = ref}
+										type="text"
+									/>
+								</Col>
+							</Row>
+							<Row>
+								<Col
+									md={8}
+									mdOffset={2}
+								>
+									<Input
+										defaultValue={initialBio}
+										label="Bio"
+										ref={(ref) => this.bio = ref}
+										type="textarea"
+									/>
+								</Col>
+							</Row>
+							{titleOptions.length > 0 &&
+								<Row>
+									<Col
+										md={4}
+										mdOffset={2}
+									>
+										<Select
+											idAttribute="unlockId"
+											label="Title"
+											labelAttribute="title"
+											options={titleOptions}
+											placeholder="Select title"
+											ref={(ref) => this.title = ref}
+										/>
+									</Col>
+								</Row>
+							}
+							<Row>
+								<Col
+									md={4}
+									mdOffset={2}
+								>
+									<SearchSelect
+										noDefault
+										collection="area"
+										defaultValue={initialArea}
+										label="Area"
+										placeholder="Select area..."
+										ref={(ref) => this.area = ref}
+										select2Options={select2Options}
+									/>
+								</Col>
+							</Row>
+							<Row>
+								<Col
+									md={4}
+									mdOffset={2}
+								>
+									<Select
+										defaultValue={initialGender}
+										idAttribute="id"
+										label="Gender"
+										labelAttribute="name"
+										options={genderOptions}
+										placeholder="Select Gender"
+										ref={(ref) => this.gender = ref}
+									/>
+								</Col>
+							</Row>
+							<Row>
+								<Col
+									md={4}
+									mdOffset={2}
+								>
+									<PartialDate
+										defaultValue={initialBirthDate}
+										label="Birth Date"
+										placeholder="YYYY-MM-DD"
+										ref={(ref) => this.birthDate = ref}
+									/>
+								</Col>
+							</Row>
+							<div className="form-group">
+								<Col
+									md={4}
+									mdOffset={4}
+								>
+									<Button
+										block
```

Comment:
Also, once the `<Col>` around this button is removed, can you remove the `block` attribute here, and add a `text-center` class to the surrounding form group div?